From d9da14bfcec0d05cfa584ea94427929c60e0cbd2 Mon Sep 17 00:00:00 2001
From: Olivier Brunel <jjk@jjacky.com>
Date: Thu, 8 Jun 2017 19:28:55 +0200
Subject: [PATCH 11/11] Fix "Date: (null)" in return receipt since 989b3a3f

A new buf_date was introduced, but the old pointer (buf) still used in
the printf()
---
 src/messageview.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/messageview.c b/src/messageview.c
index 68e90388b..01163bb7e 100644
--- a/src/messageview.c
+++ b/src/messageview.c
@@ -966,7 +966,7 @@ static gint disposition_notification_send(MsgInfo *msginfo)
 
 	/* Date */
 	get_rfc822_date(buf_date, sizeof(buf_date));
-	if (fprintf(fp, "Date: %s\n", buf) < 0)
+	if (fprintf(fp, "Date: %s\n", buf_date) < 0)
 		goto FILE_ERROR;
 
 	/* From */
-- 
2.14.1

